src/mame/drivers/hp80.cpp

```diff
@@ -108,6 +110,7 @@ class hp85_state : public driver_device
 	uint8_t m_crt_sts;
 	uint8_t m_crt_ctl;
 	uint8_t m_crt_read_byte;
+	memory_bank *m_rombank;
```

Comment:
No, this is not how to do it.  You declare the member as:
```
    required_memory_bank m_rombank;
```
And initialise it in the state class constructor with the tag to look up:
```
    m_rombank(*this, "rombank")
```